.github/workflows/mypy.yml

```diff
@@ -25,6 +25,9 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install mypy
+    - name: Install Types
+      run: |
+        python -m pip install types-requests
```

Comment:
Install necessary types. Using `mypy --install-types` does not work since it requires manual confirmation (`[yN]`).